packages/gatsby/src/commands/build.ts

```diff
@@ -23,8 +22,9 @@ import {
   showFeedbackRequest,
 } from "../utils/feedback"
 import * as buildUtils from "./build-utils"
-const { boundActionCreators } = require(`../redux/actions`)
+import { boundActionCreators } from "../redux/actions"
```

Comment:
As we're importing this, we can probably remove the `require` on line 159: https://github.com/gatsbyjs/gatsby/pull/23695/files#diff-636b7640c2360d47c928027c70960a65L157 (though please test this!)